src/Nancy/StaticContentConfigurationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Nancy
+{
+    using System.Collections.Generic;
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// Contains <see cref="StaticContentConfiguration"/> configuration extensions for <see cref="INancyEnvironment"/>.  
+    /// </summary>
+    public static class StaticContentConfigurationExtensions
+    {
+        /// <summary>
+        /// Configures <see cref="StaticContentConfiguration"/> 
+        /// </summary>
+        /// <param name="environment"><see cref="INancyEnvironment"/> that should be configured.</param>
+        /// <param name="paths">Safe paths.</param>
+        public static void StaticContent(this INancyEnvironment environment, params string[] paths)
```

Comment:
I think the `paths` parameter needs to be renamed to something more descriptive and explicit, perhaps ´safePaths` because that's what it is called everywhere else. I also think we need to update the XML comment, for the parameter, to be a lot more descriptive. The XML comment should let you understand what the parameters means without having to resort to external docks considering this is a config API